xcode/Subconscious/Shared/Services/TranscludeService.swift

```diff
@@ -11,15 +11,22 @@ import Combine
 struct Transclusion {
     let displayAddress: Slashlink
     let address: Slashlink
+    let absoluteAddress: Slashlink
```

Comment:
Should we instead store a did on the Transclusion?

Reply:
Yes, that is better.